package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "kuzzle-backoffice",
+  "name": "kuzzle-admin-console",
   "version": "2.1.5",
-  "description": "A handy administrative console for Kuzzle.io",
+  "description": "A handy administrative console for Kuzzle",
   "author": "The Kuzzle team <support@kuzzle.io>",
   "private": false,
   "bin": "./bin/kuzzle-backoffice",
```

Comment:
We should not forget to rename the binary also

Reply:
yep. didn't want to touch the program part but I can do it if there's no impact